### ISSUE TITLE
about RFN-Net

### ISSUE THREAD

**Issue by huhuchao** [hli1221/imagefusion-rfn-nest#1]
Hello, Teacher Li:
      Fortunately to read your <<RFN-Nest: An end-to-end residual fusion network for infrared and visible images >>this article. I think your work is very innovative! Here are a few questions I would like to take up your precious time to consult you.
      (1) You applied the RFN network proposed in your article to Tracking. I don't know if I understand your work correctly. You have modified the loss function of RFN, and added the loss of infrared light features to Ldetail to make the generated image easier for target detection. Then use the network trained in this way to fuse the images, and then use this image for Tracking?
     (2) The pre-training weights provided in your code are for grayscale images. Do you have pre-training weights for RGB images.
     Thank you very much for reading my email during your busy schedule. I wish you good health and success in your work!

**Comment by wanghangege** [comment]
I am not the author, for your second question, I have the same problem, and I modify the code in this repository, I find the rgb version can not be used, the only way I think, you can split rgb image to r, g, and b, than combine them after test. Have you ever complete it by another way?

**Comment by hli1221** [comment]
Hi, thanks for your attention. I am very sorry to reply you so late.
For the first question, the RFN was embedded into tracker (like Siamese RPN++) to fuse deep features, rather than fed the fused image to tracker.
For the second question, you can refer the above answer or you can train the RGB version network.
